### PR TITLE
feat(sdk): update the SDK docs domain

### PR DESCRIPTION
**Pull request by 0x6e616d**
Deployed on the repository: https://github.com/tokamak-network/thanos-sdk-docs

### CODE REVIEW

File: packages/tokamak/sdk/README.md [hunk 1]
```diff
@@ -20,16 +20,16 @@ You can find auto-generated API documentation over at [sdk.optimism.io](https://
...
 The `CrossChainMessenger` automatically connects to all relevant contracts so complex configuration is not necessary.
 
 ### L2Provider and related utilities
 
 The Optimism SDK includes [various utilities](https://github.com/tokamak-network/tokamak-thanos/blob/main/packages/tokamak/sdk/src/l2-provider.ts) for handling Optimism's [transaction fee model](https://community.optimism.io/docs/developers/build/transaction-fees/).
-For instance, [`estimateTotalGasCost`](https://sdk.optimism.io/modules.html#estimateTotalGasCost) will estimate the total cost (in wei) to send at transaction on Optimism including both the L2 execution cost and the L1 data cost.
-You can also use the [`asL2Provider`](https://sdk.optimism.io/modules.html#asL2Provider) function to wrap an ethers Provider object into an `L2Provider` which will have all of these helper functions attached.
+For instance, [`estimateTotalGasCost`](https://tokamak-network.github.io/thanos-sdk-docs/modules/index.html#estimateTotalGasCost) will estimate the total cost (in wei) to send at transaction on Optimism including both the L2 execution cost and the L1 data cost.
```
Comment (theo-learner) [inline packages/tokamak/sdk/README.md:29]: Why don't we change the connected link:
- estimateTotalGasCost: https://tokamak-network.github.io/thanos-sdk-docs/functions/l2_provider.estimateL2GasCost.html
- asL2Provider: https://tokamak-network.github.io/thanos-sdk-docs/functions/l2_provider.asL2Provider.html

**Review by theo-learner — COMMENTED** [review]
Thank you. I left a slight comment. Please check it.

File: packages/tokamak/sdk/README.md (hunk 1 above)
Comment (0x6e616d) [reply packages/tokamak/sdk/README.md]: @theo-learner thank you, I updated and redeployed at https://tokamak-network.github.io/thanos-sdk-docs/

**Review by theo-learner — APPROVED** [review]
+1

**Pull request merged**